docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -25,37 +25,37 @@ The main features of the Tizen.Content.MediaContent namespace include:
 
 - Media information
 
-    You can update the media database due to file creation, deletion, or update on the device. You can [retrieve media information](#info), add [media files and folders](#insert) to the database, and [scan for media folders](#scan).
+    You can update the media database due to file creation, deletion, or update on the device. You can [retrieve media information](#retrieving-media-information), add [media files and folders](#inserting-media-in-the-database) to the database, and [scan for media folders](#scanning-a-media-folder).
 
-    You can also retrieve [general information about the media and more specific information about the media type](#media_info).
+    You can also retrieve [general information about the media and more specific information about the media type](#media-information).
 
 - Media bookmarks
 
-    You can [insert](#inserting), [search for](#finding), and [remove](#removing) bookmarks for video and audio files.
+    You can [insert](#inserting-bookmarks), [search for](#finding-bookmarks), and [remove](#removing-bookmarks) bookmarks for video and audio files.
 
 - Media filtering
 
-    You can [create a filter](#filter) to find specific media items.
+    You can [create a filter](#setting-up-a-filter) to find specific media items.
 
 - Media playlists
 
-    You can [add](#create_playlist) or [delete](#delete_playlist) a playlist of video and audio files, and add media files to a created playlist. In addition, you can also [search for playlists](#find_playlist).
+    You can [add](#creating-playlists) or [delete](#deleting-playlists) a playlist of video and audio files, and add media files to a created playlist. In addition, you can also [search for playlists](#finding-playlists).
 
 - Media tags
 
-    You can access the tag information for the media files in the database. You can, for example, [add media tags](#tag_add), [retrieve tag information](#tag_list), and [delete tags](#tag_delete).
+    You can access the tag information for the media files in the database. You can, for example, [add media tags](#adding-tags), [retrieve tag information](#retrieving-tag-information), and [delete tags](#deleting-tags).
 
 - Media albums
 
-    You can manage an album of audio files. You can, for example, [search for albums](#findingall) and [retrieve album content](#findinginfo).
+    You can manage an album of audio files. You can, for example, [search for albums](#finding-albums) and [retrieve album content](#retrieving-album-content).
 
 - Media item groups
 
-    You can manage a collection of media items as a group, when the items have the same value of a given property. You can, for example, [search for groups](#find_groups).
+    You can manage a collection of media items as a group, when the items have the same value of a given property. You can, for example, [search for groups](#finding-media-item-groups).
 
```

Comment:
Related with above question: 

In the master branch, we have one more list item, "Media storages".
That item also has a link to the media-content.md page, not using the id generated from heading form.
Is it to keep it?

Reply:
Those are deprecated in Tizen 5.5 (https://github.com/Samsung/tizen-docs/pull/750) and it's merged tizen_5.5_prepare branch.